dev_tools/oneapi/convert_source.sh

```diff
@@ -0,0 +1,429 @@
+#!/bin/bash
+
```

Comment:
I think you should comment on what the file takes as parameter. At least, `$1`
should be a CUDA file to convert?